apps/console/src/public/deployment.config.json

```diff
@@ -242,7 +242,7 @@
                 }
             },
             "branding": {
-                "disabledFeatures": [ "branding.hostnameUrlBranding" ],
+                "disabledFeatures": [ "branding.hostnameUrlBranding", "branding.ai" ],
```

Comment:
should this be in the `branding` object, I think it would be better if we have separate object for AI.

WDYT @brionmario @chamathns @DonOmalVindula @pavinduLakshan